src/test/java/tasktypes/DeadlineTest.java

```diff
@@ -0,0 +1,13 @@
+package tasktypes;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import java.time.LocalDate;
+
+public class DeadlineTest {
+    @Test
+    public void descriptionTest() {
```

Comment:
```suggestion
    public void todoDescription_validDescription_success() {
```
`descriptionTest` is vague in terms of describing the expected input and result the test case should produce. Perhaps `todoDescription_validDescription_success` might be better?